react/features/toolbox/components/web/Toolbox.js

```diff
@@ -37,6 +36,7 @@ import { OverflowMenuItem } from '../../../base/toolbox/components';
 import { getLocalVideoTrack, toggleScreensharing } from '../../../base/tracks';
 import { isVpaasMeeting } from '../../../billing-counter/functions';
 import { VideoBlurButton } from '../../../blur';
+import { checkFilterSupport } from '../../../blur/functions';
```

Comment:
Let's give this a more specific name. A filter could potentially be anything. `checkBlurSupport`.